types/ember__test-helpers/index.d.ts

```diff
@@ -116,6 +117,10 @@ declare module '@ember/test-helpers/dom/trigger-key-event' {
     export default function(target: Target, eventType: KeyEvent, key: number | string, modifiers?: KeyModifiers): Promise<void>;
 }
 
+declare module '@ember/test-helpers/dom/triggerTab' {
+    export default function (options?: { backwards?: boolean; unRestrainTabIndex?: boolean }): Promise<void>;
```

Comment:
Let's name the function here!

```suggestion
    export default function triggerTab(options?: { backwards?: boolean; unRestrainTabIndex?: boolean }): Promise<void>;
```

That will help make autocomplete work better for consumers.